drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c

```diff
@@ -371,6 +371,116 @@ void wifi_nrf_event_proc_cookie_rsp(void *vif_ctx,
+
+void configure_tx_pwr_settings(struct nrf_wifi_tx_pwr_ctrl_params *tx_pwr_ctrl_params,
+				struct nrf_wifi_tx_pwr_ceil_params *tx_pwr_ceil_params)
+{
+	tx_pwr_ctrl_params->ant_gain_2g = CONFIG_NRF700X_ANT_GAIN_2G;
+	tx_pwr_ctrl_params->ant_gain_5g_band1 = CONFIG_NRF700X_ANT_GAIN_5G_BAND1;
+	tx_pwr_ctrl_params->ant_gain_5g_band2 = CONFIG_NRF700X_ANT_GAIN_5G_BAND2;
+	tx_pwr_ctrl_params->ant_gain_5g_band3 = CONFIG_NRF700X_ANT_GAIN_5G_BAND3;
+	tx_pwr_ctrl_params->band_edge_2g_lo = CONFIG_NRF700X_BAND_2G_LOWER_EDGE_BACKOFF;
+	tx_pwr_ctrl_params->band_edge_2g_hi = CONFIG_NRF700X_BAND_2G_UPPER_EDGE_BACKOFF;
+	tx_pwr_ctrl_params->band_edge_5g_unii_1_lo = CONFIG_NRF700X_BAND_UNII_1_LOWER_EDGE_BACKOFF;
+	tx_pwr_ctrl_params->band_edge_5g_unii_1_hi = CONFIG_NRF700X_BAND_UNII_1_UPPER_EDGE_BACKOFF;
+	tx_pwr_ctrl_params->band_edge_5g_unii_2a_lo =
+					CONFIG_NRF700X_BAND_UNII_2A_LOWER_EDGE_BACKOFF;
+	tx_pwr_ctrl_params->band_edge_5g_unii_2a_hi =
+					CONFIG_NRF700X_BAND_UNII_2A_UPPER_EDGE_BACKOFF;
+	tx_pwr_ctrl_params->band_edge_5g_unii_2c_lo =
+					CONFIG_NRF700X_BAND_UNII_2C_LOWER_EDGE_BACKOFF;
+	tx_pwr_ctrl_params->band_edge_5g_unii_2c_hi =
+					CONFIG_NRF700X_BAND_UNII_2C_UPPER_EDGE_BACKOFF;
+	tx_pwr_ctrl_params->band_edge_5g_unii_3_lo = CONFIG_NRF700X_BAND_UNII_3_LOWER_EDGE_BACKOFF;
+	tx_pwr_ctrl_params->band_edge_5g_unii_3_hi = CONFIG_NRF700X_BAND_UNII_3_UPPER_EDGE_BACKOFF;
+	tx_pwr_ctrl_params->band_edge_5g_unii_4_lo = CONFIG_NRF700X_BAND_UNII_4_LOWER_EDGE_BACKOFF;
+	tx_pwr_ctrl_params->band_edge_5g_unii_4_hi = CONFIG_NRF700X_BAND_UNII_4_UPPER_EDGE_BACKOFF;
+
+#if defined(CONFIG_BOARD_NRF7002DK_NRF7001_NRF5340_CPUAPP) || \
+defined(CONFIG_BOARD_NRF7002DK_NRF5340_CPUAPP)
+	#if DT_NODE_HAS_PROP(DT_NODELABEL(nrf70_tx_power_ceiling), max_pwr_2g_dsss)
+		tx_pwr_ceil_params->max_pwr_2g_dsss =
+			DT_PROP(DT_NODELABEL(nrf70_tx_power_ceiling), max_pwr_2g_dsss);
+	#else
+		tx_pwr_ceil_params->max_pwr_2g_dsss = 0x54;
+	#endif
+
+	#if DT_NODE_HAS_PROP(DT_NODELABEL(nrf70_tx_power_ceiling), max_pwr_2g_mcs7)
+		tx_pwr_ceil_params->max_pwr_2g_mcs7 =
+			DT_PROP(DT_NODELABEL(nrf70_tx_power_ceiling), max_pwr_2g_mcs7);
+	#else
+		tx_pwr_ceil_params->max_pwr_2g_mcs7 = 0x40;
+	#endif
+
+	#if DT_NODE_HAS_PROP(DT_NODELABEL(nrf70_tx_power_ceiling), max_pwr_2g_mcs0)
+		tx_pwr_ceil_params->max_pwr_2g_mcs0 =
+			DT_PROP(DT_NODELABEL(nrf70_tx_power_ceiling), max_pwr_2g_mcs0);
+	#else
+		tx_pwr_ceil_params->max_pwr_2g_mcs0 = 0x40;
+	#endif
+
+	#if DT_NODE_EXISTS(DT_NODELABEL(nrf70_tx_power_ceiling))
+		tx_pwr_ceil_params->rf_tx_pwr_ceil_params_override = 1;
+	#else
+		tx_pwr_ceil_params->rf_tx_pwr_ceil_params_override = 0;
+	#endif
+	tx_pwr_ceil_params->max_pwr_5g_low_mcs7 = 0;
+	tx_pwr_ceil_params->max_pwr_5g_mid_mcs7 = 0;
+	tx_pwr_ceil_params->max_pwr_5g_high_mcs7 = 0;
+	tx_pwr_ceil_params->max_pwr_5g_low_mcs0 = 0;
+	tx_pwr_ceil_params->max_pwr_5g_mid_mcs0 = 0;
+	tx_pwr_ceil_params->max_pwr_5g_high_mcs0 = 0;
+#endif
+
+#if defined(CONFIG_BOARD_NRF7002DK_NRF5340_CPUAPP)
+
```

Comment:
Please remove all newlines, they are messing up the readability

Reply:
Removed.